### ISSUE TITLE
Test failure: CTR counter function returned string not of length 16 (FreeBSD buildslave)

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7605]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| spiv reported|
|-|-|
|Trac ID|trac#384|
|Type|defect|
|Created|2003-11-05 10:42:23Z|

<details><summary>Searchable metadata</summary>

```
trac-id__384 384
type__defect defect
reporter__spiv spiv
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1068028943000000 1068028943000000
changetime__1469201088733522 1469201088733522
version__ 
owner__ 
cc__spiv cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
There's been a conch failure on the FreeBSD build slave
(http://twistedmatrix.com/buildbot/freebsd/events/1247/summary):

ERROR: testOurServerOurClient (twisted.test.test_conch.SSHTransportTestCase)
-------------------------------------------------------------------------------
Traceback: exceptions.TypeError, CTR counter function returned string not of
length 16
/home/slave/twisted/bsd-full2.2/Twisted/twisted/internet/default.py:513:_doReadOrWrite
/home/slave/twisted/bsd-full2.2/Twisted/twisted/internet/tcp.py:250:doRead
/home/slave/twisted/bsd-full2.2/Twisted/twisted/conch/ssh/transport.py:198:dataReceived
/home/slave/twisted/bsd-full2.2/Twisted/twisted/conch/ssh/transport.py:437:ssh_SERVICE_REQUEST
/home/slave/twisted/bsd-full2.2/Twisted/twisted/conch/ssh/transport.py:123:sendPacket
/home/slave/twisted/bsd-full2.2/Twisted/twisted/conch/ssh/transport.py:694:encrypt

Maybe this is related to the resolution of issue 370?
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="z3p's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|z3p commented|
|-|-|

```
#!html
<pre>
Should be fixed in t.c.s.transport:1.46
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 49e548c3db53db38c28f0e09e72a8dde39e05275
```
#!CommitTicketReference repository="" revision="49e548c3db53db38c28f0e09e72a8dde39e05275"
Merge pull request #384 from twisted/docs-source-link-8568

Author: hawkowl
Reviewer: adiroiban
Fixes: #8568, #8518
```
